File: dbconnector/src/main/java/ru/rti/kettu/sbjava/model/Song.java
package ru.rti.kettu.sbjava.model;


import org.springframework.data.mongodb.core.mapping.Document;

import javax.persistence.Column;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Document
public class Song {

    @Id
    @GeneratedValue
    private String id;
    @Column(length = 50)
    private String name;
    private Long albumId;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Long getAlbumId() {
        return albumId;
    }

    public void setAlbumId(Long albumId) {
        this.albumId = albumId;
    }
}

File: consumer/src/main/kotlin/ru/rti/kettu/sbkotlin/model/Song.kt
package ru.rti.kettu.sbkotlin.model

class Song constructor() {
    var id: String? = null
    var name: String? = null
    var albumId: Long? = null

    constructor(id: String?, name: String?, albumId: Long?) : this() {
        this.id = id
        this.name = name
        this.albumId = albumId
    }
}